Prose/ProseLib/Sources/ProseUI/BooleanIndicator.swift

```diff
@@ -0,0 +1,77 @@
+//
+// This file is part of prose-app-macos.
+// Copyright (c) 2022 Prose Foundation
+//
+
+import Assets
+import ProseCoreTCA
+import SwiftUI
+
+public struct BooleanIndicator: View {
```

Comment:
That's _really_ generic. I have no idea what a `BooleanIndicator` would look like. Also what's next? A `IntIndicator` or a `StringIndicator`? I'm a bit snarky here to press my point. Maybe the name could more reflect what the view draws instead of the underlying value. Like a LED light or something? Otherwise you could indeed call a label a `StringIndicator`.

Reply:
You're right, it's too generic. I had no other name in mind. `LEDIndicator` is a good one 🙂